recipe/run_test.py

```diff
@@ -26,14 +26,6 @@ def test_rdkit_import(self):
     import rdkit
     print(rdkit.__version__)
 
-  def test_numpy_import(self):
```

Comment:
These test are not needed because there are hard dependent packages and deepchem importing already tested.